.circleci/config.yml

```diff
@@ -81,3 +86,4 @@ workflows:
           matrix:
             parameters:
               test-type: ["not flaky", "flaky"]
+              python-version: ["3.9.2", "3.7.12"]
```

Comment:
I just checked and colab uses `3.7.13`. If images are available for this version, we could switch to it instead of `3.7.12` (but not a big deal)

Reply:
Yea great point! Looks like there is a 3.7.13 [image](https://hub.docker.com/layers/python/cimg/python/3.7.13-browsers/images/sha256-c2eb0219f868c8808c25ed93618d486c36d3a06b3f5d8551082761886168d298?context=explore). Pushing up now.

Reply:
Will these changes double the runtime of CircleCI? It is already fairly long so I think it might be overkill to run all of the tests on 3.7? Our previous issues could be caught simply trying to "import gradio" in 3.7.

Perhaps we can run just a much smaller subset of the tests in 3.7.12. WDYT?

Reply:
Good question! 

The jobs run in parallel so in theory it should not double the total time.

The latest run here took 5 minutes to finish compared to a recent commit to main that took 4 minutes.

![image](https://user-images.githubusercontent.com/41651716/179630200-7b996ad4-4912-45c2-abce-ef1d999a683c.png)

![image](https://user-images.githubusercontent.com/41651716/179630218-b4320e05-79ed-45d0-81aa-ff5c29591f4a.png)

Let me know if this time is acceptable. I can look into speeding this up but I don't think the solution will be running fewer tests. Most of the time is spent setting up the cache/building the front-end/installing ffmpeg.

Reply:
Ok great!